src/command/command.js

```diff
@@ -60,9 +60,9 @@ async function askStartQuestions() {
     return inquirer.prompt(questions);
 }
 
-function start(apiKey, wordpressUrl) {
-    if(wordpressUrl.charAt(wordpressUrl.length-1) !== '/') {
-        wordpressUrl+='/';
+const start = (apiKey, wordpressUrl) => {
```

Comment:
You have to move this before yargs.command(...), because when I've tried to use the command I've received  `ReferenceError: Cannot access 'start' before initialization` error